missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Game.java

```diff
@@ -552,6 +572,8 @@ public void startForPlayer(Player player) {
         equipmentManager.sendGameItems(player, false);
         setPlayerAttributes(player);
 
+        mwPlayer.setRandomGameEquipment(new PlayerEquipmentRandomizer(mwPlayer, this));
```

Comment:
Moved, because only from here on the arena is defined. Before that, it is not initialized in case of a map vote. The arena is needed because the RandomGameEquipment takes information from the arena config.